src/services/ReportService.js

```diff
@@ -1,4 +1,6 @@
 import Excel from "exceljs"
+import { Container } from "typedi"
+import { TechnologyService, ExecutionService } from "../services"
```

Comment:
Isso gera um warning no linter, muda pra um import separado para cada arquivo:

```suggestion
import TechnologyService from "./TechnologyService"
import ExecutionService from "./ExecutionService"
```